troubleshooting/index.md

```diff
@@ -2,101 +2,82 @@
 
 This document provides troubleshooting information for services and functionality provided by CSM.
 
-### Topics
+### Known Issues
```

Comment:
General comment:

We don't need to include the full URL for links to other markdown files in the same repository.

Reply:
Agreed and this was take under consideration was done like this on this document specific just to make sure no broken links